src/pkgcheck/checks/metadata.py

```diff
@@ -1363,13 +1382,18 @@ def __init__(self, *args, use_addon):
             r"https?://(github\.com/.*?/.*?/archive/.+\.zip|"
             r"gitlab\.com/.*?/.*?/-/archive/.+\.zip)"
         )
+        self.unstable_uris = re.compile(
+            r"^https?://files.pythonhosted.org/packages/(?:[0-9a-fA-F]{2}/){2}|"
```

Comment:
Err but these definitely aren't "unstable" — after all, they have precise checksum!

Reply:
Oh, Then I've misunderstood https://github.com/pkgcore/pkgcheck/issues/505
How should I call that url?

Reply:
removed it :)